src/SetupApi/SetupApi.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>
-    <TargetFrameworks>$(PlatformAndPortableFrameworks)</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
```

Comment:
Why is this necessary, particularly separately from #476?

Reply:
I wanted to know whether the code was running in 64-bit mode or not, so I used `Environment.Is64BitProcess`. That's not available on `netstandard1.x` so instead of using `IntPtr.Size`, I just disabled `netstandard1.x`. support. It was late 😄 .